### ISSUE TITLE
New features.

### ISSUE THREAD

**Issue by sexKAMAZ** [universalcrew/SexGuard#4]
![nza5onq9mby](https://user-images.githubusercontent.com/20995281/33526691-e18f818c-d855-11e7-882e-dd57ce979731.jpg)

- [ ] **New argument: /rg rename <name>.**
- [ ] **New argument: /rg expand <top|down> <blocks>.**
- [ ] **Particles when selecting positions.**

**Comment by Yexeed** [comment]
is moving to asynchronous sqlite still relevant?

**Comment by sexKAMAZ** [comment]
да, sqlite будет. как только у меня появится время переписать 50+% всего плагина.

**Comment by Yexeed** [comment]
ок спс
